src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -44,18 +44,18 @@ internal static class ConsolePal
 
         public static Stream OpenStandardInput()
         {
-            return new UnixConsoleStream(SafeFileHandleHelper.Open(() => Interop.Sys.Dup(Interop.Sys.FileDescriptors.STDIN_FILENO)), FileAccess.Read,
+            return new UnixConsoleStream(Interop.CheckIo(Interop.Sys.Dup(Interop.Sys.FileDescriptors.STDIN_FILENO)), FileAccess.Read,
```

Comment:
nit: I know it is an existing method, but it still looks funny because generally the convention for two-letter abbreviation is both to be capital `CheckIO` (e.g. IOException vs. LoadXml).

Reply:
I feel case-insensitive about it :smile:.